cf-custom-resources/lib/copy-assets.js

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
lol i like how succinct this lambda is

Reply:
might get more complicated with batching :)